configs/arco.json

```diff
@@ -11,7 +11,9 @@
       "tags": ["react"]
     },
     "https://arco.design/",
-    "https://arco.design/vue/docs/start"
+    "https://arco.design/vue/docs/start",
+    "https://arco.design/react/en-US/docs/start",
+    "https://arco.design/vue/en-US/docs/start"
```

Comment:
Hey, to enable filtering in your search, you can follow those small tutorials: 
- [using regular expressions](Closing since applications have been answered/deployed)
- [using custom tags](https://docsearch.algolia.com/docs/legacy/config-file/#using-custom-tags)